lib/facter/util/ip.rb

```diff
@@ -161,9 +161,9 @@ def self.get_single_interface_output(interface)
       output = Facter::Util::IP.ifconfig_interface(interface)
     when 'Linux'
       ifconfig_output = Facter::Util::IP.ifconfig_interface(interface)
-      if interface =~ /^ib/ then
+      if interface =~ /^ib.*/ then
```

Comment:
/^ib/ is equivalent to /^ib.*/ as used here, why is this change needed?

Reply:
It was a change I made when trying to figure out the problem. Since they are equivalent this line can be reverted. Thanks!

Reply:
You can revert this line since it makes no difference

Reply:
I've reverted this change.